### PR TITLE
Updates to deal with telemetry expected as array but appearing as scalars

### PR DESCRIPTION
**Pull request by menanteau**

### CODE REVIEW

File: etc/LATISS/primary_hdu.header [hunk 1]
```diff
@@ -91,24 +91,9 @@ IMAGETAG=                      / DAQ Image id
...
         ---- Information from Camera ----
-CCD_MANU=                      / CCD Manufacturer
 CCD_TYPE=                      / CCD Model Number
-CCD_SERN=                      / Manufacturers CCD Serial Number
-LSST_NUM=                      / LSST Assigned CCD Number
-SEQCKSUM=                      / Checksum of Sequencer
-SEQNAME =                      / SequenceName from Camera StartIntergration
-REBNAME =                      / Name of the REB
-CONTNUM =                      / CCD Controller (WREB) Serial Number
 TEMP_SET=                      / Temperature set point (deg C)
```
Comment (mareuter) [inline etc/LATISS/primary_hdu.header:94]: This keyword and the next are in ComCam's sensor HDU, why not put them in LATISS's?

File: etc/LATISS/primary_hdu.header [hunk 2]
```diff
@@ -91,24 +91,9 @@ IMAGETAG=                      / DAQ Image id
 OBSANNOT=                      / Observing annotation
 
         ---- Information from Camera ----
-CCD_MANU=                      / CCD Manufacturer
 CCD_TYPE=                      / CCD Model Number
```
Comment (mareuter) [inline etc/LATISS/primary_hdu.header]: Is this supposed to be duplicated as its in the senor HDU file?

**Review by mareuter — CHANGES_REQUESTED** [review]
Couple of questions about some keywords.

File: etc/LATISS/primary_hdu.header (hunk 1 above)
Comment (menanteau) [reply etc/LATISS/primary_hdu.header:94]: For LATISS, `TEMP_SET` comes a scalar from the telemetry in `ATCamera.fp_Ccd.temp`, so it stays on the common `PRIMARY` header template.

File: etc/LATISS/primary_hdu.header (hunk 2 above)
Comment (menanteau) [reply etc/LATISS/primary_hdu.header]: That was a duplicate and has been removed.

**Review by mareuter — APPROVED** [review]
Thanks for the feedback. Looks good now.

**Pull request merged**